src/test/resources/requests/rmapi/packages/post-package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "contentType": 1,
```

Comment:
can we have a string content type please?

Reply:
That's a request ot RM API, so it has to have integer content type, 
This json was inlined in a test, so I moved it to a file

Reply:
makes sense, thanks for clarifying